src/components/pages/templates-manage/TemplatesManage.js

```diff
@@ -69,8 +79,25 @@ class TemplatesManage extends React.Component {
                   </div>
                 </div>
               </div>
-            </Form>
-          </div>
+              <div className="template-fields">
+                <div className="row">
+                  <div className="column small-12 medium-8 medium-offset-2">
+                    <div className="c-question -title">
+                      <Input
```

Comment:
This input has a strange behaviour when hitting enter. Its going back to the previous screen without saving. Maybe is triggering cancel. We can prevent the default of the enter key if its not easy to solve